### ISSUE TITLE
Separate Isis Administrator Template into a Separate Package

### ISSUE THREAD

**Issue by raeldc** [joomlatools/joomla-platform#239]
What's left would just be the system template, which will have nothing but bare minimum.

**Comment by johanjanssens** [comment]
@stevenrombauts @ercanozkaya Question, should we move this or simply remove it. I don't think we really need it in platform?

**Comment by ercanozkaya** [comment]
I would remove it. No need to maintain it in a repo

**Comment by stevenrombauts** [comment]
If it's still possible to test and work on it in a clean install I'd remove it, sure.

**Comment by johanjanssens** [comment]
There wouldn't much frontend, as the system template doesn't have have module positions, and cannot show anything: https://github.com/joomlatools/joomlatools-platform/blob/master/web/templates/system/index.php

Could solve that by fixing the system index template to be the most basic possible, adding the module positions from protostar and the most bare css possible?

**Comment by stevenrombauts** [comment]
Yes, something like that sounds good to me. We need the bare minimum to test things quickly.

**Comment by johanjanssens** [comment]
Will have a look into this and see how a bare we can do.
